Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamerBase.cpp

```diff
@@ -1568,15 +1568,21 @@ void MediaPlayerPrivateGStreamerBase::emitPlayReadySession(PlayreadySession* ses
 #endif // USE(PLAYREADY)
 
 #if USE(OPENCDM)
-void MediaPlayerPrivateGStreamerBase::emitOpenCDMSession()
+CDMSessionOpenCDM* MediaPlayerPrivateGStreamerBase::getOpenCDMSession()
```

Comment:
Oh, I'm sorry because this one is my fault. WebKit guidelines specify that this kind of methods should not be called as getOpenCDMSession() but as openCDMSession()